pages/linux/tshark.md

```diff
@@ -0,0 +1,31 @@
+# tshark
+
+> Packet analysis tool, cLI version of wireshark.
+
+- Monitor everything on localhost:
+
+`tshark`
+
+- Filter captured packets by method:
```

Comment:
```suggestion
- Filter captured HTTP packets by method:
```
I would specify that it's HTTP here.

Reply:
in this example it's the http packet because we use GET method, but -Y is a filter and can be applied to anything. This works to and it's not related to HTTP: `tshark -Y 'tcp.srcport == 80'`

What do you think should I modify the example ?

Reply:
I know it's not related to HTTP, but saying "filter captured packets by method" is wrong. Packets don't have a "method"; HTTP packets do have a method.

If you want to make this more general, I would probably say something along the lines of "Only output captured packets matching a specific filter".

Reply:
```suggestion
- Only output captured packets matching a specific wireshark filter:
```